Misc/NEWS.d/next/Core and Builtins/2024-05-08-09-44-15.gh-issue-118767.iFF5F5.rst

```diff
@@ -0,0 +1,2 @@
+Using :data:`NotImplemented` in a boolean context now raises
+:exc:`TypeError`. Contributed by Jelle Zijlstra in :gh:`118767`.
```

Comment:
Isn't the link to the issue already included in the changelog?

Reply:
Thanks, fixed in the new PR.